packages/plugin-core/src/commands/SeedCommandBase.ts

```diff
@@ -38,4 +45,29 @@ export abstract class SeedCommandBase<
 
     return this.seedSvc;
   }
+
+  protected postSeedStateToWebview() {
+    const ws = getWS();
+    const existingPanel = ws.getWebView(DendronWebViewKey.SEED_BROWSER);
+
+    existingPanel?.webview.postMessage({
+      type: SeedBrowserMessageType.onSeedStateChange,
+      data: {
+        msg: this.getSeedSvc().getSeedsInWorkspace(),
+      },
+      source: "vscode",
+    });
+  }
+
+  protected async onUpdatingWorkspace() {
+    const ws = getWS();
+    await ws.updateGlobalState(
```

Comment:
for working with vscode state, can we wrap it using the state service? https://wiki.dendron.so/notes/3VvuxP7T9en9EzChUCY9D.html

Reply:
Ok - did this and then got carried away and ended up refactoring some stuff :)